### PR TITLE
New Feat/dict 54 to resolve issues with migrated properties

### PR DESCRIPTION
**Pull request by tlicht3**
In an effort to clean up the diagnosis node, I have removed the properties that have been migrated to the pathology detail node. See this node for migration details: https://jira.opensciencedatacloud.org/browse/DAT-2890

### CODE REVIEW

**Comment by ganjigajanan** [comment]
@tlicht3 `peritoneal_fluid_cytological_status` seems to be removed along with `peripancreatic_lymph_nodes_tested` could you please add it back. that will fix the failing tests.

**Comment by ganjigajanan** [comment]
Also please update the _terms.yaml file to remove any references to those deleted properties   which are specific to diagnosis node. it will not cause failure if we do not remove them, but will remain as junk otherwise.

File: gdcdictionary/examples/valid/diagnosis.json [hunk 1]
```diff
@@ -55,10 +49,7 @@
     "morphology": "8255/3",
     "ovarian_specimen_status":"Ovarian Capsule Ruptured",
     "ovarian_surface_involvement":"Absent",
-    "percent_tumor_invasion":12.3,
-    "peripancreatic_lymph_nodes_positive":"0",
-    "peripancreatic_lymph_nodes_tested":23.206363,
-    "peritoneal_fluid_cytological_status": "Atypical",
+    "peritoneal_fluid_cytological_status": "Atypical"
```
Comment (wwysoc2) [inline gdcdictionary/examples/valid/diagnosis.json:52]: Add a comma after "Atypical"

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: gdcdictionary/examples/valid/diagnosis.json (hunk 1 above)
Comment (tlicht3) [reply gdcdictionary/examples/valid/diagnosis.json]: This has been updated.

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**